src/protocols/aeternity/libs/AeternityAdapter.ts

```diff
@@ -202,9 +203,21 @@ export class AeternityAdapter extends BaseProtocolAdapter {
     rawAccount: IAccountRaw,
     idx: number,
     globalIdx: number,
-    seed: Uint8Array,
+    seed?: Uint8Array,
   ): IAccount {
-    const wallet = rawAccount.type === ACCOUNT_TYPES.hdWallet
+    if (rawAccount.type === ACCOUNT_TYPES.privateKey && rawAccount.privateKey) {
+      const address = getAddressFromPriv(Buffer.from(rawAccount.privateKey));
+      return {
+        idx,
+        globalIdx,
+        secretKey: rawAccount.privateKey!,
+        ...rawAccount,
+        privateKey: undefined,
+        address,
+        publicKey: Buffer.from(decode(address)),
+      } as IAccount;
```

Comment:
I suggest to avoid forcing the type here as it disabled type checking for particular properties.